core/ecschema-metadata/src/IncrementalLoading/IncrementalSchemaLocater.ts

```diff
@@ -14,9 +14,11 @@ import { SchemaKey } from "../SchemaKey";
 import { SchemaLoadingController } from "../utils/SchemaLoadingController";
 import { IncrementalSchemaReader } from "./IncrementalSchemaReader";
 
-interface IncrementalSchemaInfo extends SchemaInfo {
+export interface IncrementalSchemaInfo extends SchemaInfo {
```

Comment:
I'm thinking, shouldn't that rather be internal? We don't want to use it outside of itwinjs.